packages/kbn-config-schema/src/types/duration_type.test.ts

```diff
@@ -136,4 +144,6 @@ test('returns error when not string or non-safe positive integer', () => {
   expect(() => duration().validate([1, 2, 3])).toThrowErrorMatchingSnapshot();
 
   expect(() => duration().validate(/abc/)).toThrowErrorMatchingSnapshot();
+
+  expect(() => duration().validate('123foo')).toThrowErrorMatchingSnapshot();
```

Comment:
can we add one with something that would be accepted by parseInt but that should not validate, such as `123 478` ?